var/httpd/htdocs/js/Core.Agent.DynamicFieldDBSearch.js

```diff
@@ -258,13 +258,16 @@ Core.Agent.DynamicFieldDBSearch = (function(TargetNS) {
                     /TicketID=(\d+)/.exec(document.URL);
                     TicketID = RegExp.$1;
 
+                    var liElementId = $Element.parents().closest('li').attr('id');
+                    var activityDialogId = liElementId.match(/^Process_ActivityDialog-([a-f0-9]{32})$/)[1];
+
                     // serialize form
                     QueryString = Core.AJAX.SerializeForm($('#'+DynamicFieldName).closest('form'), IgnoreList) + SerializeData(UpdateList);
 
                     QueryString += ";Action="+FrontendInterface;
                     QueryString += ";Term="+encodeURIComponent(Request.term);
                     QueryString += ";MaxResults="+Core.Config.Get('Autocomplete.MaxResultsDisplayed');
-                    QueryString += ";DynamicFieldName="+encodeURIComponent(DynamicFieldName);
+                    QueryString += ";DynamicFieldName="+encodeURIComponent(DynamicFieldName.substring(0, DynamicFieldName.indexOf('_' + activityDialogId)));
```

Comment:
Have you tested whether that works for DB fields which are not part of a process? E.g. in CustomerTicketMessage, or on the agent interface? I would assume a check whether your activityDialogId exists would be good, even if just a warning is thrown.

Reply:
Indeed, this caused an error. Should be fixed with the latest commit